tests/sqlparser_common.rs

```diff
@@ -186,6 +186,94 @@ fn parse_update() {
     );
 }
 
+#[test]
+fn parse_update_set_from() {
+    let sql = "UPDATE t1 SET name = t2.name FROM (SELECT name, id FROM t1 GROUP BY id) AS t2 WHERE t1.id = t2.id";
+    let dialects = TestedDialects {
+        dialects: vec![
+            Box::new(GenericDialect {}),
+            Box::new(PostgreSqlDialect {}),
+            Box::new(BigQueryDialect {}),
+            Box::new(RedshiftSqlDialect {}),
+            Box::new(MsSqlDialect {}),
```

Comment:
@unvalley sorry I missed this, I think `Snowflake` is missing in tests, isn't it?

Reply:
It's my fault. Thank you for checking!
I added `Snowflake` in 413195b .